# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/gimp/gimpui-pc-missing-gimpmodule.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gimpui.pc.in.orig	2004-12-08 21:54:58.961259360 +0100
+++ ./gimpui.pc.in	2004-12-08 21:55:48.952659512 +0100
@@ -7,5 +7,5 @@
 Description: GIMP User Interface Library
 Version: @GIMP_REAL_VERSION@
 Requires: gimp-@GIMP_PKGCONFIG_VERSION@ gtk+-2.0 >= 2.2.0
-Libs: -L${libdir} -lgimpui-@GIMP_API_VERSION@ -lgimpwidgets-@GIMP_API_VERSION@
+Libs: -L${libdir} -lgimpui-@GIMP_API_VERSION@ -lgimpwidgets-@GIMP_API_VERSION@ -lgimpmodule-@GIMP_API_VERSION@
 Cflags: -I${includedir}/gimp-@GIMP_API_VERSION@
